co2eq/purchase/footprints.yml

```diff
@@ -7,1195 +7,1543 @@
+          - Hotel
+        source: https://github.com/tmrowco/northapp-contrib/tree/master/co2eq/purchase/exiobase
         unit: EUR
-      Liquor store:
+        coicopCode: '11.2'
+        year: 2016
+      ALCOHOLIC BEVERAGES:
         icon: wine-bottle
-        intensityKilograms: 0.8061
+        intensityKilograms:
+          AT: 0.4732692141066177
+          AU: 0.5982950629919985
+          BE: 0.5596203061793769
+          BG: 0.7036451156842851
+          BR: 0.4944338375879135
+          CA: 0.49917131766520506
+          CH: 0.2454659168135893
+          CN: 1.3468525888791105
+          CY: 0.8974388467248182
+          CZ: 0.8425858643474885
+          DE: 0.5749117505545319
+          DK: 0.6967085628784565
+          EE: 1.3140618487167322
+          ES: 0.47038763588340016
+          FI: 0.3710987330329958
+          FR: 0.4586835580872408
+          GB: 0.44238703359761267
+          GR: 0.6030354541937285
+          HR: 1.0560384144362034
+          HU: 0.8218041407136095
+          ID: 0.7019193945144384
+          IE: 0.5794624125758518
+          IN: 1.4073983442534546
+          IT: 0.4933707726549671
+          JP: 0.44284784464325566
+          KR: 0.7460210602377246
+          LT: 0.9362393641216946
+          LU: 1.057040579499641
+          LV: 0.8402462018755568
+          MT: 0.5242857225297409
+          MX: 0.6544739732540766
+          NL: 0.6653278936541067
+          NO: 0.3914318021289691
+          PL: 0.8605801438809868
+          PT: 0.5970425898062914
+          RO: 0.542272053166467
+          RU: 0.5478611192225181
+          SE: 0.26384108758934566
+          SI: 0.6834031426080678
+          SK: 0.7289731220209033
+          TR: 0.6729902757993014
+          TW: 0.7504665335086983
+          US: 0.6802822495458901
+          WA: 1.173456107467512
+          WE: 1.480603562110953
+          WF: 1.1268618640557126
+          WL: 0.8551576286106815
+          WM: 1.2659827120884126
+          ZA: 1.651660390067799
         oldKeys:
           - PURCHASE_CATEGORY_ENTERTAINMENT_LIQUOR_STORE
-          - 'Liquor Store'
-        source: 'http://www.balticproject.org/en/calculator-page'
+          - Liquor Store
+          - Liquor store
+        source: https://github.com/tmrowco/northapp-contrib/tree/master/co2eq/purchase/exiobase
         unit: EUR
-      Tobacco store:
+        coicopCode: '02.1'
+        year: 2016
+      TOBACCO:
         icon: smoking
-        intensityKilograms: 0.04627
+        intensityKilograms:
+          AT: 0.3960083585576218
+          AU: 0.7683541009074837
+          BE: 0.44399054431216184
+          BG: 0.692410699723781
+          BR: 0.44730457202390794
+          CA: 0.20966197556437272
+          CH: 0.224973909152222
+          CN: 1.98256546531643
+          CY: 0.4335260908557555
+          CZ: 0.7554379106982841
+          DE: 0.4995808550830453
+          DK: 0.314936063916008
+          EE: 0.0
+          ES: 0.3580722563818576
+          FI: 0.0
```

Comment:
0.0

Reply:
@thomasgibon, can you help us troubleshoot how these values make sense? 

If this is an anomaly, I'd suggest removing these values @corradio . No purchase has a 0 carbon footprint ;)